src/motion/__tests__/animate-prop.test.tsx

```diff
@@ -169,7 +168,7 @@ describe("animate prop as object", () => {
             }
             const Component = ({ rotate, onComplete }: any) => (
                 <motion.div
-                    animate={{ rotate }}
+                    animate={{ rotate: `${rotate}deg` }}
```

Comment:
We didn't explicitly support mixing number and string types before, this was basically a slip.